.gitignore

```diff
@@ -1 +1,4 @@
 /.idea/
+/build/reports/
+/build/resources/main/
+/build/reports/
```

Comment:
pls check merge request for extra files, it still contains .gradle folder and build/classes/java